go/vt/vtexplain/vtexplain_vttablet.go

```diff
@@ -494,16 +494,44 @@ func (t *explainTablet) HandleQuery(c *mysql.Conn, query string, callback func(*
 	// return the pre-computed results for any schema introspection queries
 	tEnv := t.vte.getGlobalTabletEnv()
 	result := tEnv.getResult(query)
-	emptyResult := &sqltypes.Result{}
-	if sidecardb.MatchesVTInitQuery(query) {
+	//emptyResult := &sqltypes.Result{}
+	fmt.Printf("query: %s\n", query)
+	/*if sidecardb.MatchesVTInitQuery(query) {
 		return callback(emptyResult)
 	}
-	if strings.HasPrefix(query, "use _vt") || strings.HasPrefix(query, "CREATE TABLE _vt") || strings.HasPrefix(query, "ALTER TABLE _vt") || strings.HasPrefix(query, "CREATE TABLE IF NOT EXISTS _vt") {
+	if //strings.HasPrefix(query, "use `fakesqldb`") ||
```

Comment:
The fakesqldb condition is commented out: intended?

Reply:
@rohit, this was commented out to show you, these changes from you PR is not needed .. But I have removed it now... You don't need to add these specific checks here instead I use existing structure of fakesqldb..